starboard/CHANGELOG.md

```diff
@@ -5,7 +5,51 @@ This file will be updated each time a new Starboard version is released.
 Each section in this file describes the changes made to the Starboard interface
 since the version previous to it.
 
-**NOTE: Starboard versions 13 and older are no longer supported.**
+**NOTE: Starboard versions 16 and older are no longer supported.**
+
+## Version 17
+Starboard 17 fully switches to POSIX APIs. The headers are
+no longer backwards compatible with older Starboard versions.
+However going forward we would continue maintaining backwards
```

Comment:
Nit, we `will continue`, not `would continue` here i think.

Reply:
Removed that part as it is confusing.